test/test_ops.py

```diff
@@ -647,6 +647,48 @@ def test_convert_boxes_to_roi_format(self):
                 self.assertTrue(torch.equal(ref_tensor, ops._utils.convert_boxes_to_roi_format(box_sequence)))
 
 
+class BoxTester(unittest.TestCase):
+    def test_bbox_xywh(self):
+        # Simple test convert boxes to xywh and back. Make sure they are same.
+        # box_tensor is in x1 y1 x2 y2 format.
+        box_tensor = torch.tensor([[0, 0, 100, 100], [0, 0, 0, 0],
+                                  [10, 15, 30, 35], [23, 35, 93, 95]], dtype=torch.float)
+        exp_xywh = torch.tensor([[0, 0, 100, 100], [0, 0, 0, 0],
+                                [10, 15, 20, 20], [23, 35, 70, 60]], dtype=torch.float)
+
+        box_xywh = ops.box_xyxy_to_xywh(box_tensor)
+        # print(box_xywh)
+        # print(box_xywh.shape)
```

Comment:
```suggestion
```

Reply:
Solved 👍

Reply:
@oke-aditya Tip for the next time: if you fully agree with a suggestion, i.e. do not want to include any other changes, you can merge it directly from the GitHub UI. If you navigate to the "Files changed" tab, you can select multiple suggestions and commit them as a batch.

Reply:
Good idea, 👍 Will try that next time. I prefer local git as I am a bit in control of what I'm doing.